g3doc/quick_reference.md

```diff
@@ -483,6 +483,10 @@ All other ops in this section are only available if `HWY_TARGET != HWY_SCALAR`:
     b[i]` for every odd `i`, in lanes `i - 1` (lower) and `i` (upper). Only
     supported if `HWY_TARGET != HWY_SCALAR`.
 
+*   `V`: `{bf,i}16`, `D`: `WidenMulPairwiseAdd<DFromV<V>>`, `VW`: `Vec<D>` \
```

Comment:
It's nicer if we define the D constraint in terms of existing operations, not the new one we are defining.
How about D: `RepartitionToWide<DFromV<V>>`? And if you like we can just inline the VW and say the function returns Vec<D> directly.

Reply:
Good spot - this was a copy-paste mistake :) thanks